lib/insights/api/common/open_api/generator.rb

```diff
@@ -441,7 +441,7 @@ def openapi_schema_properties_value(klass_name, model, key, value)
               end
 
               # Take existing attrs, that we won't generate
-              ['example', 'format', 'readOnly', 'title', 'description'].each do |property_key|
+              ['example', 'format', 'nullable', 'readOnly', 'title', 'description'].each do |property_key|
```

Comment:
Looks good, let's add a spec to cover this.

Reply:
@gmcculloug there are no specs for whole generator so it'll be quite hard to write specs for one attr

Reply:
JIRA Ticket: https://projects.engineering.redhat.com/browse/TPINVTRY-977